src/util-logopenfile.c

```diff
@@ -416,6 +417,12 @@ int SCConfLogOpenRedis(ConfNode *redis_node, LogFileCtx *log_ctx)
             SCLogError(SC_ERR_MEM_ALLOC, "Unable to allocate redis key command");
             exit(EXIT_FAILURE);
         }
+    } else if (!strcmp(redis_mode, "rlist")) {
+        log_ctx->redis_setup.command = redis_rpush_cmd;
+        if (!log_ctx->redis_setup.command) {
```

Comment:
To be honest i don't know how this can fail. I tried to match the code style around. It looks that the check was removed when the redis code moved in master to a separate file in commit a64e5e77c788b4fb05c1d67ed004cc67a725f117. maybe @fooinha can bring some light.